pages/linux/tshark.md

```diff
@@ -20,7 +20,7 @@
 
 - Specify the format of captured output:
 
-`tshark -T {{json|text|ps|…}}`
+`tshark -T {{json|text|ps}}`
```

Comment:
This should not be changed. At least keep the dots.....
```suggestion
`tshark -T {{json|text|ps|...}}`
```

Reply:
This is what I meant to do:
```suggestion
`tshark -T {{ek|fields|json|jsonraw|pdml|ps|psml|tabs|text}}`
```